mcc/capcom/capcom/capcom_test.go

```diff
@@ -230,8 +236,12 @@ func TestFindSecurityGroupsWithRange(t *testing.T) {
 			t.Error("Mismatched results and expectations length")
 		}
 		for k, v := range ret {
-			if v != tt.ret[k] {
-				t.Error("Unexpected output")
+			if v.String() != tt.ret[k].String() {
+				t.Errorf(
```

Comment:
I agree this is not the best practice we should have. I would have doubts about when the implementations of subtests would happen. I'd say when we increase the test cases, what implies a lot of other stuff, like fixutre generation helpers, and, more in general, a lot of spaceflight's tests improvements. Let's have a separate discussion on these.